src/common/types/store-data/permissions-state-store-data.ts

```diff
@@ -0,0 +1,5 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+export interface PermissionsStateStoreData {
+    hasAllPermissions: boolean;
```

Comment:
I'm not thrilled with the name... I think is missing some details, should we mention this is all-url, all-file permission we're looking for?